### PR TITLE
[Docs] How to customize the invoice logo - updated cookbook

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
| --------------- | -----
| Branch?         | 1.10 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no 
| Related tickets | https://github.com/Sylius/InvoicingPlugin/issues/254
| License         | MIT

### CODE REVIEW

**Review by lchrusciel — APPROVED** [review]

File: docs/cookbook/payments/custom-invoice.rst [hunk 1]
```diff
@@ -23,10 +23,34 @@ Before you start make sure that you have:
 #. `Sylius/InvoicingPlugin <https://github.com/Sylius/InvoicingPlugin/>`_ installed.
 #. `Wkhtmltopdf <https://wkhtmltopdf.org/>`_ package installed, because most of pdf generation is done with it.
 
+How to customize the invoice logo?
+----------------------------------------
```
Comment (CoderMaggie) [inline docs/cookbook/payments/custom-invoice.rst:27]:
```suggestion
How to change the logo in the Invoice?
--------------------------------------
```

File: docs/cookbook/payments/custom-invoice.rst [hunk 2]
```diff
@@ -23,10 +23,34 @@ Before you start make sure that you have:
...
+
+In order to change the logo on the invoice, set up ``SYLIUS_INVOICING_LOGO_FILE`` environment variable.
```
Comment (CoderMaggie) [inline docs/cookbook/payments/custom-invoice.rst:29]:
```suggestion
In order to change the logo on the invoice, set up the ``SYLIUS_INVOICING_LOGO_FILE`` environment variable.
```

File: docs/cookbook/payments/custom-invoice.rst [hunk 3]
```diff
@@ -23,10 +23,34 @@ Before you start make sure that you have:
...
+
+Example custom configuration:
+
+.. code-block:: text
+
+    SYLIUS_INVOICING_LOGO_FILE=/Users/sylius/logo_directory/your_custom_logo.png
+
+If you have a privileges issue, you may also need update ``config/packages/knp_snappy.yaml`` configuration file as well:
```
Comment (CoderMaggie) [inline docs/cookbook/payments/custom-invoice.rst:37]:
```suggestion
If you have a privileges issue, you may also need to update the ``config/packages/knp_snappy.yaml`` configuration file as well:
```

**Review by CoderMaggie — APPROVED** [review]

**Review by AdamKasp — APPROVED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "docs-cookbook-payments-custom-invoice-logo"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: docs/cookbook/payments/custom-invoice.rst [hunk 4]
```diff
@@ -23,10 +23,34 @@ Before you start make sure that you have:
...
+How to change the logo in the Invoice?
+----------------------------------------
+
+In order to change the logo on the invoice, set up the ``SYLIUS_INVOICING_LOGO_FILE`` environment variable.
+
+Example custom configuration:
+
+.. code-block:: text
+
+    SYLIUS_INVOICING_LOGO_FILE=/Users/sylius/logo_directory/your_custom_logo.png
```
Comment (GSadee) [inline docs/cookbook/payments/custom-invoice.rst:35]: You should put a custom logo in the project directory and then use it with the proper path, so as to fix the problem from issue: https://github.com/Sylius/InvoicingPlugin/issues/254

**Review by GSadee — CHANGES_REQUESTED** [review]

File: docs/cookbook/payments/custom-invoice.rst [hunk 5]
```diff
@@ -23,10 +23,27 @@ Before you start make sure that you have:
...
+
+Example custom configuration:
+
+.. code-block:: text
+
+    SYLIUS_INVOICING_LOGO_FILE=%kernel.project_dir%/assets/custom-logo.png
+
+Make sure to clear the cache each time the configuration is changed.
+
+If you have a permission issue when generating an invoice, you may also need to update the ``config/packages/knp_snappy.yaml`` configuration file.
```
Comment (GSadee) [inline docs/cookbook/payments/custom-invoice.rst:39]: Could you give an example with what should be set?

File: docs/cookbook/payments/custom-invoice.rst [hunk 6]
```diff
@@ -23,10 +23,37 @@ Before you start make sure that you have:
...
+
+.. code-block:: yaml
+
+    knp_snappy:
+    pdf:
+        enabled:    true
+        binary:     '%env(WKHTMLTOPDF_PATH)%'
+        options:
+            allow: 'path-to-the-logo-directory'
```
Comment (GSadee) [inline docs/cookbook/payments/custom-invoice.rst:48]:
```suggestion
    knp_snappy:
        pdf:
            options:
                allow: %kernel.project_dir%/assets/custom-logo.png
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Rafał! :1st_place_medal:

**Pull request merged**